src/Libraries/CoreNodes/List.cs

```diff
@@ -441,9 +441,9 @@ public static IDictionary Deconstruct(IList list)
         ///     Sort list based on its keys
         /// </summary>
         /// <param name="list">list to be sorted</param>
-        /// <param name="keys">list of keys</param>
-        /// <returns name="sorted list">sorted list</returns>
-        /// <returns name="sorted keys">sorted keys</returns>
+        /// <param name="keys">list of keys, keys have to be sortable (ex: numbers,strings)   </param>
+        /// <returns name="sortedList">List sorted using keys</returns>
+        /// <returns name="sortedKeys">Keys sorted</returns>
```

Comment:
To me this seems unnecessary, `Sorted list` and `Sorted keys` is very clear.

Reply:
This is to keep consistency with the rest of the outputs (camelCase)
Maybe the description is a bit obvious.

Reply:
Yea not talking about the names, that makes sense. Just think the new description are unnecessary.

Reply:
addressed in 172ba45